site_scons/prereq_tools/base.py

```diff
@@ -523,8 +523,7 @@ def __init__(self, env, opts):
 
     def run_build(self, opts):
         """Build and dependencies"""
-        # argobots is not really needed by client but it's difficult to separate
-        common_reqs = ['ucx', 'ofi', 'hwloc', 'mercury', 'boost', 'uuid', 'crypto', 'protobufc',
+        common_reqs = ['ofi', 'hwloc', 'mercury', 'boost', 'uuid', 'crypto', 'protobufc',
```

Comment:
shouldn't we also remove ucx references in utils/build.config ?

Reply:
may as well also remove the recipe in site_scons/components/__init__.py too?

Reply:
Yes, I suppose so.  Will do.